server/test_nlp_manual.py

```diff
@@ -1,202 +0,0 @@
-# Copyright 2022 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-"""
-Script to test flow of nlp features.
-
-How to run locally:
-FLASK_ENV=local python test_nlp_manual.py
-"""
-from dataclasses import dataclass
```

Comment:
curious if you meant to delete these tests because they test the nl data API (i.e., /api/nl/data endpoint) which is used by the regular nl interface?

Reply:
this is a very old manual test that we don't use anymore.